Src/ILGPU/IndexTypes.tt

```diff
@@ -690,4 +695,4 @@ namespace ILGPU
     }
 
 <#  } #>
-}
```

Comment:
ISSUE: Please remove this change.

We currently enforce that T4 should not end with a newline on its own. This is because there are known issues with transforming T4 templates on other platforms.

Reply:
This is probably your text editor. It has been configured to save and ensure there is a blank newline. Try disabling the setting, or using a different text editor to make the change. Alternatively, you can use git to remove the change.

Reply:
I'm using github's web editor. I looked for a setting, but couldn't find anything. I don't know how to use a different editor on github, or git itself.

Reply:
I couldn't find any solution when using Github's web editor.

I've raised a PR into your fork of ILGPU:
https://github.com/Nnelg/ILGPU/pull/1

If you accept the PR, it should fix the newline issue.

Reply:
I want to hold off on this until we're sure the rest of the code is good, because any time I edit it'll be put right back in...